package.json

```diff
@@ -41,38 +41,37 @@
     "primeng": "^10.0.0",
     "quill": "^1.3.7",
     "rxjs": "~6.5.5",
-    "systelab-components": "^11.0.0",
-    "systelab-login": "^9.0.0",
-    "systelab-preferences": "^7.0.0",
-    "systelab-translate": "^9.0.0",
+    "systelab-components": "^13.2.6",
+    "systelab-login": "^11.1.0",
+    "systelab-preferences": "^9.0.0",
+    "systelab-translate": "^11.0.0",
     "tslib": "^2.0.0",
-    "zone.js": "~0.10.3"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.1001.2",
-    "@angular-devkit/build-ng-packagr": "^0.1001.2",
-    "@angular/cli": "^10.1.2",
-    "@angular/compiler-cli": "^10.1.2",
-    "@angular/language-service": "^10.1.2",
-    "@types/jasmine": "~3.5.9",
+    "@angular-devkit/build-angular": "^12.2.13",
+    "@angular/cli": "^12.2.13",
+    "@angular/compiler-cli": "^12.2.13",
+    "@angular/language-service": "^12.2.13",
+    "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.8",
     "@types/node": "^12.12.62",
     "allure-js-commons": "^1.3.2",
-    "codelyzer": "^5.1.2",
+    "codelyzer": "^6.0.0",
     "jasmine-allure-reporter": "^1.0.2",
-    "jasmine-core": "~3.5.0",
-    "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~4.3.0",
+    "jasmine-core": "~3.6.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "^6.3.8",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage-istanbul-reporter": "~2.1.0",
```

Comment:
Use 'karma-coverage' because 'karma-coverage-istanbul-reporter' usage has been deprecated since version 11.
Refer to https://github.com/systelab/systelab-components/issues/413